### PR TITLE
Packaging & testday updates

### PR DESCRIPTION
**Pull request by pmkovar**
* Update packaging navigation
* Update testday instructions

See also #699 and #609.

### CODE REVIEW

File: source/.htaccess.haml [hunk 1]
```diff
@@ -414,6 +414,7 @@ directory_index: false
   Redirect 301 /case-studies/osas-test-lab /user-stories
   Redirect 301 /case-studies /user-stories
   Redirect 301 /rdo-manager /tripleo
+  Redirect 301 /documentation/packaging /documentation/rdo-packaging
```
Comment (apevec) [inline source/.htaccess.haml]: Can we also add Redirect for old URL /packaging/ to redirect here?
I got reports some folks had it bookmarked in the past and now it's 404.

File: source/documentation/packaging.html.md [hunk 2]
```diff
@@ -1,37 +0,0 @@
...
-upstream code packaged for Red Hat Enterprise Linux, CentOS and Fedora.
-
-
-<div class="alert alert-info">
-[Red Hat](http://redhat.com) also provides
-[Red Hat OpenStack Platform](https://www.redhat.com/en/technologies/linux-platforms/openstack-platform)
-which is based on *RDO*.
-</div>
-
-## Relation to upstream
```
Comment (apevec) [inline source/documentation/packaging.html.md]: Where will this part fit, do we have it already elsewhere?

File: source/.htaccess.haml (hunk 1 above)
Comment (pmkovar) [inline source/.htaccess.haml]: Good idea, will do.

File: source/documentation/packaging.html.md (hunk 2 above)
Comment (pmkovar) [inline source/documentation/packaging.html.md]: On the front page, we also say:

> RDO is a community of people using and deploying OpenStack on CentOS, Fedora, and Red Hat Enterprise Linux. 

> If you are looking for enterprise-level support, or information on partner certification, Red Hat also offers Red Hat OpenStack Platform.

Do we want to be more specific there?

**Pull request merged**